<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 150px; 
}</string>
    </property>
    <property name="text">
     <string>Update Information</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastNameInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
